packages/@ama-sdk/core/src/clients/api-angular-client.ts

```diff
@@ -63,7 +63,7 @@ export class ApiAngularClient implements ApiClient {
     let opts = options;
     if (this.options.requestPlugins) {
       for (const plugin of this.options.requestPlugins) {
-        opts = await plugin.load().transform(opts);
+        opts = await plugin.load({logger: this.options.logger}).transform(opts);
```

Comment:
could be simplified to

```suggestion
        opts = await plugin.load(this.options).transform(opts);
```